### ISSUE TITLE
[BUG]: Exception thrown when using 'is defined' inside {{ }} in Volt

### ISSUE THREAD

**Issue by sheroesde** [phalcon/cphalcon#14476]
In Volt (using `Phalcon\Mvc\Volt\Simple`) I sometimes using shorthand notation for IF/ELSE like:

```
{{ someVar is defined ? 'yes' : 'no' }}
```

In Phalcon 3.4 all is fine, but it is broken in 4.0.0-rc.1, where I get

```
Phalcon\Mvc\View\Exception: Syntax error, unexpected token IDENTIFIER(is) in /home/my-project/app/views/premium.volt
```

**Comment by zsilbi** [comment]
Confirmed, although it does work within the `{% %}` tags, for example:

`{% set myVar = someVar is defined ? 'yes' : 'no' %}`

**Comment by Jurigag** [comment]
Well as i see @sergeyklay was doing some changes to volt parsers etc lately so maybe it was some of his change?

**Comment by elcreator** [comment]
same for me

{{ uploadedPhoto is user.avatar ? 'checked="checked"' : '' }}

Syntax error, unexpected token IDENTIFIER(is) in /....../views/registered/widgets/avatar.twig on line 31

**Comment by Jeckerson** [comment]
@elcreator What version of Phalcon do you use?

**Comment by elcreator** [comment]
Version | 4.0.0-rc.3
Build Date | Nov 16 2019 20:03:22
Powered by Zephir | Version 0.12.10-fedc314
PHP Version | 7.2.24-1

**Comment by elcreator** [comment]
as temporary fix I have replaced "is" with "=="

**Comment by mvanvu** [comment]
The same for me ` {{ title is empty ? siteName : title }}` => Syntax error, unexpected token IDENTIFIER(is) ...

Version | 4.0.0-rc.3
Php 7.3

**Comment by madrian-es** [comment]
What's the status on this? We have a lot of code written for v4.0.0-rc.1 and we can't update to the latest version since this will imply rewriting A LOT of code.

Do you expect this to be fixed any time soon, or should we go ahead and rewrite everything?

**Comment by sergeyklay** [comment]
@madrian-es I'll try to sort out ASAP

**Comment by sergeyklay** [comment]
Status update: I fixed the issue. Thinking about better internal implementation

**Comment by sergeyklay** [comment]
Fixed in  #14909
